Frameworks/include/CGGradientInternal.h

```diff
@@ -34,3 +34,11 @@ class __CGGradient : private objc_object {
     void initWithColorComponents(const float* components, const float* locations, size_t count, CGColorSpaceRef colorspace);
     void initWithColors(CFArrayRef components, const float* locations, CGColorSpaceRef colorspace);
 };
+
+unsigned long _CGGradientGetCount(CGGradientRef gradient);
+
+float* _CGGradientGetStopLocation(CGGradientRef gradient);
+
+float* _CGGradientGetColorComponents(CGGradientRef gradient);
```

Comment:
don't much like returning raw float* here.  I think this entire class could use some updating